src/modules/components/odsModulesProvider/odsModulesProvider.mdx

```diff
@@ -15,7 +15,11 @@ wrapped using the following providers:
 -   `<WagmiProvider />`: needed to know which chain and RPC endpoint to use when fetching blockchain data;
 -   `<QueryClientProvider />`: needed to cache the data fetched from the blockchain;
 
-The provider also renders the `<OdsCoreProvider />` and passes the values set in the `coreProviderValues` property.
+The provider also renders the `<OdsCoreProvider />` and passes the values set in the `coreProviderValues` property and
+the values set in the `values` property for module-specific configurations.
```

Comment:
Could you improve the readability here? Sounds like the provider passes the `values` prop to the `OdsCoreProvider`. I would document the module-specific props in a new sentence

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/f666bcf9caf68ae81d6d59c9342bf52a57d03f2d